oopsla2024-benchmarks/Cargo.toml

```diff
@@ -7,9 +7,9 @@ publish = false
 [dependencies]
 arbitrary = "1.3"
 average = "0.14"
-cedar-policy-core = { git = "https://github.com/cedar-policy/cedar-spec", branch = "release/3.0.x", version = "=3.0.1" }
-cedar-policy-generators = { git = "https://github.com/cedar-policy/cedar-spec", branch = "release/3.0.x", version = "=3.0.1" }
-cedar-policy-validator = { git = "https://github.com/cedar-policy/cedar-spec", branch = "release/3.0.x", version = "=3.0.1" }
+cedar-policy-core = { path = "cedar-spec/cedar/cedar-policy-core", version = "=4.0.0" }
+cedar-policy-generators = { path = "cedar-spec/cedar-policy-generators", version = "=4.0.0" }
+cedar-policy-validator = { path = "cedar-spec/cedar/cedar-policy-validator", version = "=4.0.0" }
```

Comment:
Since this is going to `main`, the intent was to pull the "latest" version. Since we've bumped to 4.1.0 on `main` now, I'll move to that. Alternatively: is there a way to get "whatever version is at that path"?

Reply:
`version = "*"`

Reply:
Aha, makes sense. Updated in the latest commit.